src/UnitTests/General.cs

```diff
@@ -3,11 +3,41 @@
 using Shouldly;
```

Comment:
Revert changes to this file.

Reply:
Apologies - that was a hangover from testing another reported issue simultaneously.  Removed now.